client/core/core.go

```diff
@@ -1207,6 +1210,42 @@ func (c *Core) handleUnbookOrderMsg(dc *dexConnection, msg *msgjson.Message) err
 	return ob.Unbook(&note)
 }
 
+// handleEpochOrderMsg is called when an epoch_order notification is
+// received.
+func (c *Core) handleEpochOrderMsg(dc *dexConnection, msg *msgjson.Message) error {
+	var note msgjson.EpochOrderNote
+	err := json.Unmarshal(msg.Payload, &note)
+	if err != nil {
+		return fmt.Errorf("epoch order note unmarshal error: %v", err)
+	}
+
+	// Ensure the associated dex lists the market referenced by the epoch note.
+	match := false
+	for _, mkt := range dc.markets {
+		if mkt.Display() == note.MarketID {
```

Comment:
Will this work? For a dcr/btc market, `Display` will return **DCR-BTC**, but I believe that market name will be set by the server as **dcr_btc**.

Reply:
making the epoch queue a field of the order book simplifies this bit since the order book can be looked up with the market id provided by the note, the associated epoch queue can then be referenced.